src/typings/config.ts

```diff
@@ -36,6 +36,7 @@ export interface DashkitGroupRenderProps {
     editMode: boolean;
     items: ConfigItem[];
     layout: ConfigLayout[];
+    context: any;
```

Comment:
No other variants?

Reply:
Not really for now, only one is to rewrite all types to Generic